server/log.js

```diff
@@ -0,0 +1,15 @@
+const conf = require('./config.js');
+
+let notLocalHost =
+  conf.env === 'production' &&
+  conf.s3_bucket !== 'localhost' &&
+  conf.bitly_key !== 'localhost';
```

Comment:
This snippet seems to be in 3-4 places. May want to refactor it out into a utils file or something.